src/Agent.Worker/Handlers/Helpers/OutputDataHelper.cs

```diff
@@ -0,0 +1,18 @@
+using System.Text.RegularExpressions;
+
+namespace Agent.Worker.Handlers.Helpers;
+
+public static class OutputDataHelper
+{
+    public static string RemoveAnsiColorsFromLine(string input)
+    {
+        if (string.IsNullOrWhiteSpace(input))
+        {
+            return input;
+        }
+
+        return AnsiColorsRegex.Replace(input, string.Empty);
+    }
+
+    private static readonly Regex AnsiColorsRegex = new(@"\u001b\[[^m]*m", RegexOptions.Compiled);
```

Comment:
I think this is a scenario where we should have a telemetry-only mode so we can evaluate the impact before rolling it out.  

Things to collect:
1. Maybe: Performance impact (e.g. get sampling stop watch for time spent in RemoveAnsiColorsFromLine) e.g. for every X (1000?  10000?) calls, measure time spent in replace using System.Daignostics.StopWatch
2. Length of strings matched